pages/flow/[id].js

```diff
@@ -118,21 +121,29 @@ export default function FlowPage({ asanas, currentFlowDB }) {
           <StyledForm onSubmit={(event) => onSubmitDescription(event)}>
             <StyledTextArea
               rows="4"
-              cols="40"
+              cols="30"
               id="description"
               name="description"
               aria-label="add description for flow"
               placeholder="add your description..."
               defaultValue={flow.description}
+              onChange={(event) => handleMaxLength(event.target.value)}
             />
+            {disabledSaveButton && (
+              <StyledWarning>
+                The maximum length of the description is 100 characters.
+              </StyledWarning>
+            )}
 
             <MainButton
               type="primary"
-              width="5rem"
+              width="7rem"
               aria-label="Save description"
+              disabled={disabledSaveButton ? true : false}
```

Comment:
Why you make a statement here. Just use this:
```suggestion
              disabled={disabledSaveButton}
```

Reply:
Done, thanks :)